tests/ubuntu_test.yaml

```diff
@@ -0,0 +1,21 @@
+schemaVersion: '1.0.0'
```

Comment:
this needs to be `2.0.0` I believe. the original schema version didn't support the `args` field.

Reply:
Does 2.0.0 work inside of bazel?  @dlorenc had commented that the schema version might need to be v1.0.0 to work with bazel:
https://github.com/GoogleCloudPlatform/debian-docker/pull/98#discussion_r143586836

Reply:
done